web/components/comments-list.tsx

```diff
@@ -20,12 +16,20 @@ import { LoadingIndicator } from './loading-indicator'
 
 const COMMENTS_PER_PAGE = 50
 
-type ContractComment = Comment & { contractId: string }
+type ContractComment = Comment & {
```

Comment:
Would it be worth moving this type into `comment.ts` and exporting it for others to use?

Reply:
This is basically analogous to our contracts and the typing "should" be designed in the same way, but I don't know whether any other code really would care about having more precise typing on this right now. I'll fix it up if you like.

Reply:
I'm going to leave this for a later PR since it's kind of reaching into stuff that is out of scope for this one, but I will do it.